Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -40,6 +42,7 @@ import PayPalCheckout
         orderCreationClient.createOrder(with: request) { [weak self] result in
             switch result {
             case .success(let order):
+                self?.apiClient.sendAnalyticsEvent("ios.paypal-native-checkout.create-order.started")
```

Comment:
Also, it's ideal that we have the same analytic logging b/w platforms per payment method feature/flow. Not just in terms of naming, but also in terms of timing of analytic sendoff in the flow.

This PR adds more events than currently exist for Android NXO flow. This is a good opportunity to make sure we are tracking the same events across both platforms.

Reply:
Yeah I think we need a follow up ticket on the Android side to add additional events. Right now, we don't track success or failure at all for NXO on Android, just which flow was started. 🙈 I'll create an Android follow up ticket for an audit on that platform for NXO analytics events.

Reply:
Good spot!